app/components/frame-viewer.cjsx

```diff
@@ -193,23 +195,24 @@ module.exports = React.createClass
     unless @state.panEnabled
       @setState panEnabled: true, =>
         this.refs.subjectImage.focus()
-  
-  togglePanOff: ->
-    @setState panEnabled: false
-  
+
+  togglePanOff: (e) ->
+    unless e.relatedTarget?.title is "zoom out" or e.relatedTarget?.title is "zoom in"
```

Comment:
Checking title attributes seems fragile here (I'm thinking what if the titles get deleted or changed). Could you use refs on the buttons and compare DOM nodes? eg. `e.relatedTarget == @refs.zoomOutButton` or something similar?

I'm not sure if `event.relatedTarget` is supported in Firefox, so that might need checking.